test/pipe.js

```diff
@@ -50,23 +50,54 @@ assert.equal(shell.error(), null);
 assert.equal(result.toString(), shell.cat('resources/uniq/pipeSorted').toString());
 
 // Synchronous exec
-// TODO: add windows tests
-if (process.platform !== 'win32') {
+if (process.platform === 'win32') {
+  // Windows-specific
+
+  // Ensure the user does not have the wrong version of FIND. I.e.,
+  // require the following to return error.
+  result = shell.exec('CD..&FIND . -name no');
```

Comment:
Sorry, I didn't explain myself clearly. I mean, use `var find = shell.which('find')` to get a path to find. Now replace `CD..&FIND` with the `find` variable.

You'll still need to check to make sure it's the right `find`, but it should still guarantee that it's not referring to the local `find.js`. `CD..&FIND` is very confusing to non-Windows people, whereas `shell.which()` more intuitively returns a path to `find`.

Or if there's so much trouble with the `find` command on Windows, what if there's a more straightforward alternative?